modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -216,21 +223,21 @@ sudo ./gravity upgrade --resume
 +
 . The upgrade procedure streams output to your terminal session. If the previous error reoccurs, perform the troubleshooting steps described in the following section.
  
-== Troubleshooting Component-Level (Appliance) Upgrade Errors
+== Troubleshooting Appliance Upgrade Errors
 
-A specific sequence of steps is performed during a cluster-level component (installer package) upgrade. If an error occurs, the upgrade pauses and displays an error. In most cases, the availability of running applications is not impacted when running multiple replicas of each application on a production Runtime Fabric configuration.
+A specific sequence of steps is performed during an appliance (installer package) upgrade. If an error occurs, the upgrade pauses and displays an error. In most cases, the availability of running applications is not impacted when running multiple replicas of each application on a production Runtime Fabric configuration.
```

Comment:
Intro text is longer and more obvious than it needs to be. Consider something like this:
```suggestion
If an error occurs, the upgrade pauses and displays the reason. In most cases, the availability of multiple running applications on a production Runtime Fabric configuration is not impacted.
```